messaginginapp/src/main/java/io/customer/messaginginapp/gist/data/listeners/Queue.kt

```diff
@@ -24,7 +24,6 @@ import retrofit2.converter.gson.GsonConverterFactory
 class Queue : GistListener {
 
     private var localMessageStore: MutableList<Message> = mutableListOf()
-    private val cacheMap = mutableMapOf<String, Response>()
```

Comment:
@Shahroz16 just to confirm, this is not needed right?

Reply:
@BernardGatt yes!! feel free to remove it.